docs/reference-guide/metadata.md

```diff
@@ -0,0 +1,81 @@
+---
+id: metadata
+title: Metadata
+sidebar_label: Metadata
+---
+
+Bolt protocol specifies additional data that can be sent along with the
+requested results. Such data is called metadata and can be divided into two
+groups:
+ - Query Statistics,
+ - Notifications.
+
+Both of these metadata can be accessed through `summary` map that is being sent
+along with the results of the query. Query statistics will be under `stats` key,
+and notifications under `notifications` key.
+
+## Query Statistics
+
+Query statistics will be sent whenever a user performs a query that will effect
+data in any way. In other words we will track the quantity of these changes
+throughout the query execution and report it back to the user.
+
+The structure of statistics is a map of string keys and integer values. Data
+that is being tracked:
+
+ - `nodes-created`
+ - `nodes-deleted`
+ - `relationships-created`
+ - `relationships-deleted`
+ - `labels-added`
+ - `labels-removed`
+ - `properties-set`
+
```

Comment:
Please add a sentence or two about the differences between these counters and the triggers.

Reply:
Ohh, actually I was thinking about a bit different thing. When a user executes a query like `CREATE (n) DELETE n;`, then both `nodes-created` and `nodes-deleted` will be `1`, but no trigger will be triggered, because the triggers accumulates the changes throughout a transaction, while these counters aren't. In my opinion this might be a bit confusing for the users.

But your comment is also true and important, I just didn't think about it before, so I think this is the first time when miscommunication resulted in something better :smile:

Reply:
Aha ok that makes sense, I will use your example to communicate that better